Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBoxRig.cs

```diff
@@ -15,7 +15,7 @@ public class BoundingBoxRig : MonoBehaviour
         [Header("Flattening")]
         [SerializeField]
         [Tooltip("Choose this option if Rig is to be applied to a 2D object.")]
-        private BoundingBox.FlattenModeEnum flattenedAxis;
+        private BoundingBox.FlattenModeEnum flattenedAxis = default(BoundingBox.FlattenModeEnum);
```

Comment:
None?

Reply:
I'm not sure what this means. Are you saying to explicitly set this to a specific default value?

![image](https://user-images.githubusercontent.com/3580640/40800172-759ac6fc-64c4-11e8-9b7b-23da32beba79.png)

Reply:
Hmm, should def have a none option on that enum.

vNEXT I suppose.

Reply:
`DoNotFlatten` is kinda a `None`, but that name change would make it more obvious.